app/models/user.rb

```diff
@@ -17,15 +17,24 @@ def remember
     update_attributes remember_digest: User.digest(remember_token)
   end
 
-  def authenticated? remember_token
-    return false if remember_digest.nil?
-    BCrypt::Password.new(remember_digest).is_password? remember_token
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
```

Comment:
send là gì em nhỉ ?

Reply:
send la 1 method 
chung ta da dinh nghia 1 attribute  bang  1 :activation
theo day send de send 1 attribute
send co the send theo cach noi suy cac doi tuong de su dung tuong ung